fast64_internal/sm64/sm64_objects.py

```diff
@@ -2166,12 +2179,24 @@ def draw_level_path(self, layout):
         return True
 
     def draw_actor_path(self, layout):
-        actor_path = Path(bpy.context.scene.fast64.sm64.decomp_path) / "actors"
-        if not filepath_ui_warnings(layout, (actor_path / self.actor_group_name).with_suffix(".c")):
+        if self.export_locations is None:
             return
```

Comment:
```suggestion
            return False
```
and other returns in this function
would be cleaner